README.md

```diff
@@ -72,6 +78,9 @@ Belle is friends with Mrs. Potts
 Write your code here:
 ```ruby
 # code here
+friends.each do |friend|
+  p "Belle is friends with #{friend}"
```

Comment:
This is fine, but I'd caution of `p` vs `puts`. If you're interested, I'd checkout this [link](http://stackoverflow.com/questions/1255324/p-vs-puts-in-ruby)